packages/graz/src/types/wallet.ts

```diff
@@ -48,10 +56,7 @@ export const WALLET_TYPES = [
   WalletType.OKX,
 ];
 
-export type Wallet = Pick<
-  Keplr,
-  "enable" | "getOfflineSigner" | "getOfflineSignerAuto" | "getOfflineSignerOnlyAmino" | "signDirect" | "signAmino"
-> & {
+export type Wallet = Pick<Keplr, "enable" | "getOfflineSignerOnlyAmino" | "signAmino"> & {
```

Comment:
let's not override getOfflineSigner, getOfflineSignerAuto, signDirect

Reply:
I use the same structure as Keplr type, but using the `OfflineAminoSigner` & `OfflineDirectSigner` directly from `@cosmjs`, because Keplr type still use Long instead of BigInt